platform/default/include/mbgl/storage/offline_schema.hpp

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 // THIS IS A GENERATED FILE; EDIT offline_schema.sql INSTEAD
-// To regenerate, run `node platform/default/mbgl/storage/offline_schema.js`
+// To regenerate, run `node platform/default/include/mbgl/storage/offline_schema.sql`
```

Comment:
@tmpsantos should you be editing a generated file? 🤔

Reply:
Also, why the change from .js to .sql?

Reply:
Ops, last one was a typo. The change was made by generator..